dwave/cloud/testing/mocks.py

```diff
@@ -223,6 +223,44 @@ def qpu_pegasus_solver_data(m: int,
 
     return structured_solver_data(**params)
 
+
+def qpu_zephyr_solver_data(m: int,
+                           t: int = 4,
+                           **kwargs) -> dict:
+    """Mock QPU solver data with a custom-sized Zephyr topology.
+
+    Args:
+        m:
+            Grid parameter for the Zephyr lattice.
+        t:
+            Tile parameter for the Zephyr lattice.
+        **kwargs:
+            Solver properties passed down to :meth:`.structured_solver_data`.
+    """
+    try:
+        import dwave_networkx as dnx
+    except ImportError:     # pragma: no cover
+        raise RuntimeError("Can't generate Zephyr graph without dwave-networkx. "
+                           "Install with 'dwave-cloud-client[mocks]'.")
+
+    graph = dnx.zephyr_graph(m, t)
+    qubits = sorted(list(graph.nodes))
```

Comment:
Why sort here? Looks like Zephyr nodes are ordered already

Reply:
Yes, as are Pegasus nodes. But not Chimera.. So, just to ensure qubits and couplers are ordered like they are on SAPI (future-proof).